### PR TITLE
feat: embed playground in the presentation page

### PR DESCRIPTION
**Pull request by Odonno**
image

### CODE REVIEW

File: src/layouts/Layout.astro [hunk 1]
```diff
@@ -29,6 +29,7 @@ const { title } = Astro.props
         <nord-layout>
             <NavGroup />
             <nord-header slot="header">
+                {/* TODO : to replace by a search bar on desktop */}
```
Comment (amiceli) [inline src/layouts/Layout.astro:32]: I use only nord component, you can check this component : https://nordhealth.design/components/command-menu/

Comment (amiceli) [reply src/layouts/Layout.astro:32]: I will do it after this PR if you wnat.

**Comment by amiceli** [comment]
I'm checking UI before merge.

File: src/layouts/Layout.astro (hunk 1 above)
Comment (Odonno) [reply src/layouts/Layout.astro:32]: Yes. You can use `pagefind` for example to create a search index from Astro files.

**Comment by amiceli** [comment]
It's ok, I merge and deploy it.

**Pull request merged**